readme.md

```diff
@@ -0,0 +1,43 @@
+# Austin Journal Club Blog
```

Comment:
Actually, you don't have to add this readme any more because we have https://austinmljournalclub.github.io/quarto.html

So I suggest you to just leave this file out from the PR

Reply:
ok, I think I've fixed this

Reply:
and I've fixed it such that it isn't dependent on the other PR now, sorry about that